smart-contracts/package.json

```diff
@@ -43,6 +43,7 @@
     "hardlydifficult-eth": "1.1.4",
     "hardlydifficult-ethereum-contracts": "0.11.1",
     "lint-staged": "11.2.6",
+    "node-fetch": "^3.1.0",
```

Comment:
used?

Reply:
yes used to fetch data from the graph in the `lock:managers` task

Reply:
replaced by cross-fetch btw